dune_api_scripts/store_query_result_for_todays_trading_data.py

```diff
@@ -14,7 +14,7 @@
     dune = dune_from_environment()
 
     # fetch query result id using query id
-    query_id = int(os.getenv('QUERY_ID_TODAYS_TRADING_DATA', "135804"))
+    query_id = int(os.getenv('QUERY_ID_TODAYS_TRADING_DATA', "249240"))
```

Comment:
Why did this change?

Reply:
This was not intended in this PR.
 But 135804 is no longer the newest one. In staging, we only update 249240.